v3/src/components/graph/adornments/movable-line.scss

```diff
@@ -10,9 +10,10 @@
 }
 
 .movable-line-handle {
+  display: none;
   fill: #fff;
   height: 12px;
-  pointer-events: all;
+  pointer-events: none;
```

Comment:
This makes the line's drag handles draggable. See PT comment [here](https://www.pivotaltracker.com/story/show/181923044/comments/237515910). Since the fix was so simple, I snuck it into this PR.